### DISCUSSION TITLE
Import / Export options

### DISCUSSION THREAD

**Discussion by bsaunder2002** [jithware/brethap#88] [Ideas]
It's nice that I can create a breathing pattern / change a lot of options in the Preferences page.
I think it would be great if I could export (email?) my set of preference patterns.  Then others could import them.
If we looked at a bunch of different science studies / historical breathing exercises, we could develop a library of them.
This coincides with supporting more than 5 patterns.
As well as an easier way to change patterns from the main scree.

**Comment by jithware** [comment]
I do like this suggestion. Converting this discussion to a feature request.

**Reply by jithware** [reply]
See issue https://github.com/jithware/brethap/issues/90

**Comment by jithware** [comment]
> As well as an easier way to change patterns from the main scree.

See [this](https://github.com/jithware/brethap#running) for preference selection on home screen
